presto-main/src/main/resources/webapp/assets/utils.js

```diff
@@ -128,7 +128,8 @@ function formatState(state, fullyBlocked) {
 }
 
 function getHostname(url) {
-    var hostname = new URL(url).hostname;
+    var urlObj = new URL(url);
+    var hostname = urlObj.hostname + ":" + urlObj.port;
     if ((hostname.charAt(0) == '[') && (hostname.charAt(hostname.length - 1) == ']')) {
```

Comment:
Since the port is now included, the brackets for IPv6 must stay, so this needs to be removed